freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/test/model/CheckGroupsRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.test.model;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("CheckGroupsRequest")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CheckGroupsRequest extends ClientTestBase {
+
+    @NotNull
```

Comment:
`@NotNull` is included in `@NotEmpty`, isn't it?

Reply:
yes you're right